webview/src/commonMain/kotlin/com/multiplatform/webview/cookie/CookieManager.kt

```diff
@@ -0,0 +1,18 @@
+package com.multiplatform.webview.cookie
+
+
+/**
+ * Cookie Manager exposing access to cookies of the WebView.
+ * This is an interface to allow platform specific implementations.
+ * ---------------------------------------------------------------
+ * PS: Not having it as expect/actual class was a conscious decision,
+ * since expect/actual classes will be marked as beta in coming kotlin releases.
+ * */
+interface CookieManager {
+    suspend fun setCookie(url: String, cookie: Cookie)
+    suspend fun getCookies(url: String): List<Cookie>
+    suspend fun removeAllCookies()
```

Comment:
We should also have a delete cookie function for a URL.

`suspend fun removeCookies(url: String)`

Reply:
Also `WebSettings` should have following options.
```

  - isCookiesEnabled: Boolean
  - isThirdPartyCookiesEnabled: Boolean
```

Reply:
Great progress overall, kudos @KevinnZou